UKHomeOffice/ide-controller
# COP-2062 Clear all data

## Description

In this PR
- Add a function to clear all context api (Data Store)
- Link this funciton to `Clear all data` button
- Add timer to clear data after 15mins

## Testing
- `npm i && npm run dev` or `npm i && npm build`
- Run the application
- Scan a document
- Wait 15mins
- Check data still exists
- Take picture
- Wait 15mins
- Check data

### Expected results
- The data should be removed


## Test 2
- `npm i && npm run dev` or `npm i && npm build`
- Run the application
- Scan and/ or allow the cam to take picture
- Click `Clear all data` button

### Expected results
- All data should be removed
  - Scanned document
  - Chip image
  - Live photo



## Developer Checklist

\* Required
